Source/diablo.cpp

```diff
@@ -427,6 +427,7 @@ static void SaveOptions()
 	setIniInt("Graphics", "Gamma Correction", sgOptions.Graphics.nGammaCorrection);
 	setIniInt("Graphics", "Color Cycling", sgOptions.Graphics.bColorCycling);
 	setIniInt("Graphics", "FPS Limiter", sgOptions.Graphics.bFPSLimit);
+	setIniInt("Graphics", "Gradual Durability Warning", sgOptions.Graphics.bDurabilityIconGradual);

```

Comment:
Not a fan of this setting. My vote is still to leave it out.

Reply:
I'm personally indifferent to this. Surely in the long run we'll have an ingame menu for these options, and then I'm more okay with the ability to tweak minor things.

Reply:
> Surely in the long run we'll have an ingame menu for these options

For this one, I think an option in the menu is _also_ not warranted. If I believed this should be a setting on the menu, I'd be ok with it being a setting on the ini.